support/rollup/examples.js

```diff
@@ -0,0 +1,66 @@
+import path from 'path'
+import alias from 'rollup-plugin-alias'
+import resolve from 'rollup-plugin-node-resolve'
+import commonjs from 'rollup-plugin-commonjs'
+import json from 'rollup-plugin-json'
+import replace from 'rollup-plugin-replace'
+import babel from 'rollup-plugin-babel'
+import uglify from 'rollup-plugin-uglify'
+import sourcemaps from 'rollup-plugin-sourcemaps'
+import pkg from '../../package.json'
+
+const configurations = []
+
+const umdConfig = {
+  input: 'examples/index.js',
+  output: {
+    file: pkg.browser,
+    name: 'slate-examples',
+    format: 'umd',
+    exports: 'named',
+    sourcemap: process.env.ROLLUP_WATCH,
+  },
+  plugins: [
+    alias({
+      debug: path.resolve(__dirname, 'node_modules', 'debug', 'src', 'browser'),
+      'react-dom/server': path.resolve(__dirname, 'node_modules', 'react-dom', 'cjs', 'react-dom-server.browser.production.min'),
+    }),
+    resolve(),
```

Comment:
I think actually it would be nice to have a comment on top of each of the Rollup plugins quickly summarizing what it does / why it's needed. It's hard to come back to these kinds of files later and understand them.